docs/general/ad-filtering/create-own-filters.md

```diff
@@ -792,6 +792,12 @@ The rule corresponds to requests for media files — music and video, e.g. `.mp4
 
 The rule corresponds to browser plugins resources, e.g. Java or Flash.
 
+:::info Compatibility
+
+Rules with `$object` modifier are not supported by AdGuard for Safari and AdGuard for iOS and have limited CoreLibs support.
```

Comment:
the table should be updated
https://adguard.com/kb/general/ad-filtering/create-own-filters/#content-type-modifiers
https://uploads.adguard.org/slztQeERSBKqzpw6Kc.png